docs/src/pages/components/slider/CustomizedSlider.js

```diff
@@ -78,7 +78,7 @@ const IOSSlider = withStyles({
     boxShadow: iOSBoxShadow,
     marginTop: -14,
     marginLeft: -14,
-    '&:focus,&:hover,&$active': {
+    '&:focus,&:hover,&:active': {
```

Comment:
This references the `active` class defined here not the `:active` pseudo class.

https://github.com/mui-org/material-ui/pull/16630/files#diff-e0045e743b6d722a7b3842c0191b561dL89

Reply:
Oh, ok, now it makes sense. I thought there should be `:active` pseudo class